PennMobile.xcodeproj/project.pbxproj

```diff
@@ -2922,14 +2920,13 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = navigation;
-				CF_BUNDLE_LONG_VERSION_STRING = "$(CF_BUNDLE_LONG_VERSION_STRING)";
 				CF_BUNDLE_SHORT_VERSION_STRING = "$(CF_BUNDLE_SHORT_VERSION_STRING)";
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = PennMobile/PennMobile.entitlements;
 				CODE_SIGN_IDENTITY = "Apple Development";
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				CODE_SIGN_STYLE = Manual;
-				CURRENT_PROJECT_VERSION = "$(CF_BUNDLE_LONG_VERSION_STRING)";
+				CURRENT_PROJECT_VERSION = "$(CF_BUNDLE_SHORT_VERSION_STRING)";
```

Comment:
Idk if this matters for anything, but if `CURRENT_PROJECT_VERSION` is supposed to be unique, this could somehow mess up since `CF_BUNDLE_SHORT_VERSION_STRING` isn't necessarily unique.